sections/03_categorize.md

```diff
@@ -14,8 +14,8 @@ positive to have disease categories changed by data, or would the changing
 definition (i.e. as more data are accumulated) actually be harmful? What impacts
 would this have on the training of physicians?*
 
-*What are the major challenges in this space, and does deep learning enable us to
-tackle any of them? Are there example approaches whereby deep learning is
+*What are the major challenges in this space, and does deep learning enable us
+to tackle any of them? Are there example approaches whereby deep learning is
 already having a transformational impact? I (Casey) have added some sections
 below where I think we could contribute to the field with our discussion.*
```

Comment:
I just came across this: http://www.cs.ucf.edu/~bagci/publications/DLradiologyRSNA2016.pdf
Will see if worth trying to incorporate anything from it